pkg/controller/pattern1/deployment.go

```diff
@@ -433,7 +433,8 @@ func DashboardDeployment(apimanager *apimv1alpha1.APIManager,y *configvalues, nu
 					Containers: []corev1.Container{
 						{
 							Name:  "wso2am-pattern-1-analytics-dashboard",
-							Image: "wso2/wso2am-analytics-dashboard:3.0.0",
+							// Image: "wso2/wso2am-analytics-dashboard:3.0.0",
+							Image: "pubudu/wso2am-analytics-dashboard:3.1.0-rc1",
```

Comment:
Image location shall we use the WSO2 repository.

Reply:
Used only for pre-release testing. Fixed in latest code.